src/main/java/org/folio/service/impl/EcsTlrServiceImpl.java

```diff
@@ -35,4 +35,24 @@ public EcsTlr post(EcsTlr ecsTlr) {
     return requestsMapper.mapEntityToDto(ecsTlrRepository.save(
       requestsMapper.mapDtoToEntity(ecsTlr)));
   }
+
+  @Override
+  public boolean put(UUID requestId, EcsTlr ecsTlr) {
+    log.debug("put:: requestId: {}, ecsTlr: {}", requestId, ecsTlr);
+
+    return ecsTlrRepository.findById(requestId)
+      .map(ecsTlrEntity -> requestsMapper.mapDtoToEntity(ecsTlr))
+      .map(ecsTlrRepository::save)
+      .isPresent();
```

Comment:
I think we should choose an approach here and stick to it. Method `get()` returns an `Optional`, and controller converts this `Optional` into a response. We should probably do the same in your two methods. Or don't return `Optional` in any of the methods.

Also, if you fetch the request by ID only to make sure that it exists, take a look at repository's method `existsById()`.